prime-router/src/main/resources/db/migration/V39__delivery_statistics.sql

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
master currently has another migration as V39, so this causes conflicts when attempting to migrate

Reply:
Thanks, Luis!  If that's the only problem, can you go ahead and mark your change as approved?

Our practice is to do a final build locally before the final merge, so that catches these flyway conflicts.